docs/core/compatibility/interop/9.0/cet-support.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Breaking change: CET supported by default"
+description: Learn about the breaking change in interop in .NET 8 where CET is supported by default.
+ms.date: 10/04/2024
+---
+# CET supported by default
+
+`apphost` and `singlefilehost` are now marked as Intel CET&mdash;compatible (they're compiled with the `/CETCOMPAT` option). This change was made to enhance security of .NET applications. However, it imposes a limitation on the shared libraries that .NET apps can load and interop with. Libraries aren't allowed to set thread context to a location with an instruction pointer that's not present on the shadow stack or in a table of allowed continuation addresses for exception handling.
+
+## Previous behavior
+
+Previously, shared libraries loaded into the .NET process were able to set thread context using <xref:Microsoft.VisualStudio.CorDebugInterop.ICorDebugProcess.SetThreadContext(System.UInt32,System.UInt32,System.IntPtr)>, `RtlRestoreContext/NtContinue`, or their exception handlers to any location in the process address space.
```

Comment:
Is this the correct SetThreadContext API?

Reply:
No, this one is a debugger API. The correct one is https://learn.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-setthreadcontext.